buffalo/cmd/generate/action.go

```diff
@@ -0,0 +1,141 @@
+// Copyright © 2016 NAME HERE <EMAIL ADDRESS>
```

Comment:
@apaganobeleno whatever is at the top of the other ones? or delete it. don't really care either way. you know what, just delete it. we have a license covering the project. don't need another one there.